Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM docker.elastic.co/logstash/logstash-oss:7.1.1
+FROM docker.elastic.co/logstash/logstash-oss:7.15.2
```

Comment:
@chennv4 did you test this docker image and verified it works with tls?

Reply:
Tested with letsencrypt TLS enabled cluster and it works fine.